app/src/androidTest/java/com/github/multimatum_team/multimatum/SignInActivityTest.kt

```diff
@@ -37,8 +38,8 @@ class SignInActivityTest {
 
     @Before
     fun init() {
-        hiltRule.inject()
         Intents.init()
+        hiltRule.inject()
```

Comment:
Is there really an importance to swap these two lines ?

Reply:
Other test cases were working correctly with the calls in this order, so I used the same order here to make sure that this does not cause any issue. So it probably changes nothing, but I think it is better to keep it like that just in case.